domain/secretbackend/doc.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package secretbackend provides domain types and services for managing secret
+// backend definitions and model-level backend selection.
+//
+// The secret backend domain is responsible for:
```

Comment:
"is responsible for" requires -ing verbs, and -ing verbs are longer, more abstract, and harder to process. Unless it's critical to emphasize "is responsible for", and to distinguish what it should do from what it actually does (should be the same, but who knows), I'd simplify:

The secret backend domain:
- defines and validates...
- creates, updates, lists, ..